### DISCUSSION TITLE
Pinterest Rich Pin metadata tag

### DISCUSSION THREAD

**Discussion by mezotv** [vercel/next.js#70867] [Ideas]
### Goals

1. Adding support for the rich pin meta tag
2. Make it future proof so additional pinterest tags can be added

### Background

I was looking at the tool https://socialsharepreview.com/ and saw their pinterest section. For pinterests rich pin feature they have a way to opt pages in and out. This feature should allow the user to do that. In the future there might be additional meta tags for pinterest so having this cant hurt.

### Proposal

I have already made a draft pr for this. It works similar to facebook or twitter in terms of usability.

As mentioned I already have a draft pr for this that I quickly made. https://github.com/vercel/next.js/pull/70873

It would work something like this:

```js
pinterest: {
   richPin: true
  }
  ```

 And would resolve to:
 
```html
 <meta property="pinterest-rich-pin" content="true" />
 ```

**Comment by dpaulos6** [comment]
This sound like a good thing to implement and very easy to use! 👍

**Reply by mezotv** [reply]
Yea I was looking to implement that tag in my app, but using nextjs metadata feature would be a bit easier and more streamlined.

**Comment by mezotv** [comment]
This has been added in 75831c08e83abd120be3968ec32ee0e2a2c4d33d!
